src/WaterPipe/WaterPipe.php

```diff
@@ -154,6 +154,14 @@ class WaterPipe
      */
     private $_mapRegistry;
 
+    /**
+     * In case an exception occures during pipe execution, it will be
+     * stored here for user code to read in middlewares and errorhandlers
+     *
+     * @var Exception
```

Comment:
```suggestion
     * @var \Exception
```

Intellisense can break here for some IDE without the backslash for native, not imported (there is no `use Exception` in the file), PHP classes